app/components/Inputs/Label/Label.jsx

```diff
@@ -0,0 +1,14 @@
+// @flow
+import React from 'react'
+import styles from './Label.scss'
+
+type Props = {
+  label: string
+}
+
+export default class Label extends React.Component<Props> {
+  render() {
+    const { label } = this.props
+    return <label className={styles.label}> {label} </label>
```

Comment:
Perhaps better with text as child instead of string property?
```
return <label className={styles.label}>{this.props.children}</label>
```

Reply:
I decided to keep the text prop and pass children in for the additional content after reviewing the alternative this seemed to make the most sense and the easiest for styling/layout4